### PR TITLE
Enhance some op error message

### PR DESCRIPTION
**Pull request by LielinJiang**
Enhance error message of Normal, Uniform, roi_perspective_transform, locality_aware_nms, Categorical, MultivariateNormalDiag, mine_hard_examples.

### CODE REVIEW

File: paddle/fluid/operators/detection/mine_hard_examples_op.cc [hunk 1]
```diff
@@ -165,85 +165,107 @@ class MineHardExamplesOp : public framework::OperatorWithKernel {
...
 
     auto cls_loss_dims = ctx->GetInputDim("ClsLoss");
     auto idx_dims = ctx->GetInputDim("MatchIndices");
     auto dis_dims = ctx->GetInputDim("MatchDist");
 
-    PADDLE_ENFORCE_EQ(cls_loss_dims.size(), 2UL,
-                      "The shape of ClsLoss is [N, Np].");
+    PADDLE_ENFORCE_EQ(
+        cls_loss_dims.size(), 2UL,
+        platform::errors::InvalidArgument("The shape of ClsLoss is [N, Np]."));
```
Comment (ceci3) [inline paddle/fluid/operators/detection/mine_hard_examples_op.cc]: need to print cls_loss_dims.size() according to the request

Comment (LielinJiang) [reply paddle/fluid/operators/detection/mine_hard_examples_op.cc]: done, thanks.

**Comment by LielinJiang** [comment]
image
image
image

image
image
image
image
image

**Review by Boyan-Liu — APPROVED** [review]

**Review by ceci3 — APPROVED** [review]
LGTM

**Pull request merged**